packages/twenty-front/src/modules/object-record/record-show/hooks/useRecordShowContainerTabs.ts

```diff
@@ -145,7 +145,7 @@ export const useRecordShowContainerTabs = (
         tabs: {
           workflow: {
             title: 'Flow',
-            position: 0,
+            position: 101,
```

Comment:
Do as with the other object specific layouts: position it after the fields in the side panel.